docs/Contributing/Release.md

```diff
@@ -0,0 +1,143 @@
+# Release Process
+
+
+## Prerequisites:
+- Prepare release notes
+
+
+## Weekly chores
+### Enarx dependency update
+- Update local code
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
+  ```
+- Checkout chore branch
+  ```bash
+  git checkout -b update-prerequisites
+  ```
+- Check to see `Cargo.toml` is refering to the latest updated prerequisite crates
+- Run cargo-update on all individual sub-crates in updated packages are referenced
+   ```bash
+   for d in internal/*/ ; do (cd "$d" && cargo update); done
+   ```
+- Git commit sub-crate update
+  ```bash
+   git commit -asm 'Updated Enarx sub-crate dependencies'
+   ```
+- Run cargo update on `enarx`
+  ```bash
+  cargo update
+  ```
+- Git commit sub-crate update
+  ```bash
+   git commit -asm 'Updated Enarx dependencies'
```

Comment:
idem, this should follow the commit guidelines we agreed upon and subject should be in imperative tense